src/test/regress/expected/single_node.out

```diff
@@ -2279,6 +2278,219 @@ SELECT pg_reload_conf();
+);
+CREATE TABLE dist_table_1(
+    dist_col integer,
+    int_col integer,
+    text_col_1 text,
+    text_col_2 text
+);
+SELECT create_distributed_table('dist_table_1', 'dist_col');
+ create_distributed_table
+---------------------------------------------------------------------
+
+(1 row)
+
+INSERT INTO dist_table_1 VALUES (10, 11, 'string_10', 'string_11');
+NOTICE:  executing the command locally: INSERT INTO single_node.dist_table_1_102048 (dist_col, int_col, text_col_1, text_col_2) VALUES (10, 11, 'string_10'::text, 'string_11'::text)
+-- test remote execution
+INSERT INTO local_table_4
+SELECT
+    t1.dist_col,
+    1,
+    'string_1',
+    'string_2',
+    2,
+    'string_3',
+    t1.text_col_1,
+    t1.text_col_2
+FROM dist_table_1 t1
+RETURNING *;
+ col_1 | col_2 |  col_3   |  col_4   | col_5 |  col_6   |   col_7   |   col_8
+---------------------------------------------------------------------
+    10 |     1 | string_1 | string_2 |     2 | string_3 | string_10 | string_11
+(1 row)
+
+-- test local execution
+BEGIN;
+    INSERT INTO local_table_4
+    SELECT
+        t1.dist_col,
+        3,
+        'string_4',
+        'string_5',
+        4,
+        'string_6',
+        t1.text_col_1,
+        t1.text_col_2
+    FROM dist_table_1 t1
+    RETURNING *;
+NOTICE:  executing the command locally: SELECT dist_col, 3, 'string_4'::text AS col_3, 'string_5'::text AS col_4, 4, 'string_6'::text AS col_6, text_col_1, text_col_2 FROM single_node.dist_table_1_102048 t1 WHERE true
+NOTICE:  executing the command locally: SELECT dist_col, 3, 'string_4'::text AS col_3, 'string_5'::text AS col_4, 4, 'string_6'::text AS col_6, text_col_1, text_col_2 FROM single_node.dist_table_1_102049 t1 WHERE true
+NOTICE:  executing the command locally: SELECT dist_col, 3, 'string_4'::text AS col_3, 'string_5'::text AS col_4, 4, 'string_6'::text AS col_6, text_col_1, text_col_2 FROM single_node.dist_table_1_102050 t1 WHERE true
+NOTICE:  executing the command locally: SELECT dist_col, 3, 'string_4'::text AS col_3, 'string_5'::text AS col_4, 4, 'string_6'::text AS col_6, text_col_1, text_col_2 FROM single_node.dist_table_1_102051 t1 WHERE true
+ col_1 | col_2 |  col_3   |  col_4   | col_5 |  col_6   |   col_7   |   col_8
+---------------------------------------------------------------------
+    10 |     3 | string_4 | string_5 |     4 | string_6 | string_10 | string_11
+(1 row)
+
+COMMIT;
+-- test EXPLAIN (ANALYZE)
+EXPLAIN (ANALYZE, COSTS OFF, SUMMARY OFF, TIMING OFF)
+INSERT INTO local_table_4
+SELECT
+    t1.dist_col,
+    5,
+    'string_7',
+    'string_8',
+    6,
+    'string_9',
+    t1.text_col_1,
+    t1.text_col_2
+FROM dist_table_1 t1
+RETURNING *;
+                                  QUERY PLAN
+---------------------------------------------------------------------
+ Insert on local_table_4 (actual rows=1 loops=1)
+   ->  Custom Scan (Citus Adaptive) (actual rows=1 loops=1)
+         Task Count: 4
+         Tuple data received from nodes: 54 bytes
```

Comment:
TODO: hmmm this is flaky, is there any way to normalize those lines ?

Reply:
hmm, might be because citus.enable_binary_protocol is false before pg14 by default

Reply:
but we're already doing something that, weird
https://github.com/citusdata/citus/blob/master/src/test/regress/sql/single_node.sql#L7-L8

Reply:
but at some point, we're doing "\c - - - coordinator", so need to set this GUC to on one more time